lib/srv/sess.go

```diff
@@ -734,13 +735,28 @@ func (s *session) getNamespace() string {
 	return s.registry.srv.GetNamespace()
 }
 
-// pollAndSync is a loop inside a goroutite which keeps synchronizing the terminal
-// size to what's in the session (so all connected parties have the same terminal size)
-// it also updates 'active' field on the session.
+// pollAndSync is a loops forever trying to sync terminal size to what's in
+// the session (so all connected parties have the same terminal size) and
+// update the "active" field of the session. If the session are recorded at
+// the proxy, then this function does nothing as it's counterpart in the proxy
+// will do this work.
 func (s *session) pollAndSync() {
 	log.Debugf("[session.registry] start pollAndSync()\b")
```

Comment:
while we are on it, can you remove `\b` and user log.WithFields? otherwise it renders strange in logs